### PR TITLE
Touchscreen fix

### PR DESCRIPTION
**Pull request by breycarr**
Touchscreen functionality now maintained when app is running

### CODE REVIEW

File: src/controller.py [hunk 1]
```diff
@@ -6,12 +6,3 @@ def welcome(self):
     def soil_moisture(self, plant_class):
         plant = plant_class()
         return plant.soil_moisture()
-
```
Comment (olliebrownlow) [inline src/controller.py:9]: Do we still need a controller class?

File: src/web/index.html [hunk 2]
```diff
@@ -3,17 +3,18 @@
...
-    <script type='text/javascript' src='/main.js'></script>
-    
+    <script type='text/javascript' src='eel.js'></script>
+    <script src="https://cdn.plot.ly/plotly-latest.min.js"></script>
+    <script type='text/javascript' src='main.js'></script>
+
 </head>
-<body>
+<body bgcolor="black" id='body'>
+    <center>
```
Comment (olliebrownlow) [inline src/web/index.html:12]: A center tag!! never seen that - I love it and will use it often!

File: src/controller_test.py [hunk 3]
```diff
@@ -24,33 +24,3 @@ def soil_moisture(self):
                 return 900
```
Comment (olliebrownlow) [inline src/controller_test.py]: Up at the top of this file, do we still need a MockLight class now we don't test for Lights?

**Review by olliebrownlow — COMMENTED** [review]
Great!

File: src/controller.py.save [hunk 4]
```diff
@@ -0,0 +1,32 @@
+import time
```
Comment (t-ekla) [inline src/controller.py.save]: Why does the file name end in .save?

File: src/web/main.js [hunk 5]
```diff
@@ -4,16 +4,19 @@ var interval, cnt, startButton, stopButton
...
 }], {
   yaxis: {
     range: [300, 1030]
-  }
-});
+  },
+  height:200,
+  width:450,
+  margin: {l: 35, r: 35, t: 10, b: 20},
+}, {displayModeBar: false});
```
Comment (t-ekla) [inline src/web/main.js:19]: what's displayModeBar?

**Review by t-ekla — APPROVED** [review]
Looks great!

File: src/controller.py (hunk 1 above)
Comment (breycarr) [reply src/controller.py:9]: No! We can probably remove it from the app!

File: src/controller_test.py (hunk 3 above)
Comment (breycarr) [reply src/controller_test.py]: We don't even need the controller, so we removed them!

**Pull request merged**